runtime/compiler/env/J9SharedCache.cpp

```diff
@@ -897,11 +907,11 @@ TR_J9SharedCache::rememberClass(J9Class *clazz, const AOTCacheClassChainRecord *
       return chainData;
       }
 
-   int32_t numSuperclasses = TR::Compiler->cls.classDepthOf(fe()->convertClassPtrToClassOffset(clazz));
-   int32_t numInterfaces = numInterfacesImplemented(clazz);
+   int32_t numSuperclasses = fe()->numSuperclasses(clazz);
+   int32_t numInterfaces = fe()->numInterfacesImplemented(clazz);
```

Comment:
Since `numSuperclasses` and `numInterfaces` are used a little later on to fill in the class chain, I got rid of the `necessaryClassChainLength` call instead.